### PR TITLE
feat : added landing page

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (45)
Screenshot (46)
Screenshot (47)

added langing page, not responsive

### CODE REVIEW

**Review by zabihshaik — APPROVED** [review]
Everything looks good.

File: index.html [hunk 1]
```diff
@@ -1,13 +1,145 @@
...
+        <div class="text">
+          <h1 class="text-title">pick the best for your children</h1>
+        </div>
+        <button class="text-btn">shop now</button>
+      </div>
+    </main>
+    <h2 class="title">categories</h2>
+    <div class="categories">
+      <div class="category">
+        <img src="/assets/batmanCar2.png" class="category-image" alt="" />
```
Comment (chanchal-panpaliya) [inline index.html]: alt is blank

**Review by chanchal-panpaliya — APPROVED** [review]
looks good to merge

File: index.html (hunk 1 above)
Comment (Sharmaryan) [reply index.html]: fixed

**Pull request merged**